ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/fees/EIP1559.java

```diff
@@ -61,7 +61,7 @@ public long computeBaseFee(
       final BigInteger target = BigInteger.valueOf(targetGasUsed);
       final BigInteger denominator = BigInteger.valueOf(feeMarket.getBasefeeMaxChangeDenominator());
       feeDelta = pBaseFee.multiply(gDelta).divide(target).divide(denominator).longValue();
-      baseFee = parentBaseFee - feeDelta;
+      baseFee = max(parentBaseFee - feeDelta, 0);
```

Comment:
I am not certain if this should be 0 or 7 here.  I think a basefee under 7 is pathological

Reply:
I removed this part after a specification update from Micah